README.md

```diff
@@ -8,8 +8,9 @@ The Google API client library for .NET enables access to Google APIs such as Dri
 Supported Platforms
 =================================
 
-* .NET Framework 4.5
-* Windows Store apps
+* .NET Framework 4.5 and 4.6
+* .NET Core (via netstandard1.3 support)
+* Windows 8 Apps
```

Comment:
I think highlighting that we don't support it might be a good idea.   
[Universal Windows Platform App with google calendar](http://stackoverflow.com/q/40522651/1841839)   I actually didn't know we didn't support it.  Do we have plans ti supporting it at some point?